rero_ils/modules/documents/jsonschemas/documents/document_subjects-v0.0.1.json

```diff
@@ -1,24 +1,362 @@
+              "title": "Type",
+              "type": "string",
+              "default": "bf:Place",
+              "const": "bf:Place",
+              "form": {
+                "wrappers": [
+                  "hide"
+                ]
+              }
+            },
+            "preferred_name": {
+              "$ref": "#/preferred_name"
+            },
+            "identifiedBy": {
+              "$ref": "#/identifiedBy"
+            },
+            "source": {
+              "$ref": "#/source"
+            }
+          }
+        },
+        {
+          "$ref": "https://ils.rero.ch/schemas/documents/document_subjects_links-v0.0.1.json#/temporal"
+        },
+        {
+          "title": "Time-span",
+          "type": "object",
+          "additionalProperties": false,
+          "propertiesOrder": [
+            "type",
+            "term",
+            "identifiedBy",
+            "source"
+          ],
+          "required": [
+            "type",
+            "term"
+          ],
+          "properties": {
+            "type": {
+              "title": "Type",
+              "type": "string",
+              "default": "bf:Temporal",
+              "const": "bf:Temporal",
+              "form": {
+                "wrappers": [
+                  "hide"
+                ]
+              }
+            },
+            "term": {
+              "title": "Term",
+              "type": "string",
+              "minLength": 3
+            },
+            "identifiedBy": {
+              "$ref": "#/identifiedBy"
+            },
+            "source": {
+              "$ref": "#/source"
+            }
+          }
         }
-      }
+      ]
     },
     "form": {
       "hide": true,
       "navigation": {
         "essential": true
       }
     }
+  },
+  "preferred_name": {
+    "title": "Preferred name",
+    "type": "string",
+    "minLength": 3
+  },
+  "source": {
+    "title": "Source",
+    "description": "Source of the subject, for instance MeSH, rero or IdRef.",
+    "type": "string",
+    "default": "rero",
+    "enum": [
+      "MeSH",
+      "rero",
+      "IdRef"
+    ],
+    "form": {
+      "Type": "selectWithSort",
+      "options": [
+        {
+          "label": "MeSH",
+          "value": "MeSH"
+        },
+        {
+          "label": "rero",
+          "value": "rero"
+        },
+        {
+          "label": "IdRef",
+          "value": "IdRef"
+        }
+      ]
+    }
+  },
+  "identifiedBy": {
+    "title": "Identifier",
+    "type": "object",
+    "additionalProperties": false,
+    "propertiesOrder": [
+      "type",
+      "source",
+      "value"
+    ],
+    "required": [
+      "value"
+    ],
+    "properties": {
+      "type": {
+        "title": "Type",
+        "type": "string",
+        "default": "bf:Local",
+        "const": "bf:Local",
+        "form": {
+          "options": [
+            {
+              "value": "bf:Local",
+              "label": "bf:local"
+            }
+          ],
```

Comment:
Can you add in the enum: "bf:Local", "rero-rameau", "gnd".
No default value is set.